addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/protocol/APICommand.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.dscalarm.internal.protocol;
+
+/**
+ * Class to create a API Command string.
+ *  
+ * @author Donn Renk - (https://code.google.com/p/homewatcher/)
```

Comment:
WARNING: This project is licensed under GPL, which is incompatible with the EPL, i.e. we MUST NOT include any such code in openHAB. Please either:
- rewrite the code yourself
- get approval by the original author that he is ok to contribute this code under EPL to openHAB. Then add an Also-By footer to your commit record (see https://wiki.eclipse.org/Development_Resources/Contributing_via_Git#The_Commit_Record)